box/tarantool_test.go

```diff
@@ -67,6 +69,407 @@ func TestBox_Info(t *testing.T) {
 	validateInfo(t, resp.Info)
 }
 
+func TestBox_Sugar_Schema_UserCreate(t *testing.T) {
+	const (
+		username = "exists"
+		password = "exists"
+	)
+
+	ctx := context.TODO()
+
+	conn, err := tarantool.Connect(ctx, dialer, tarantool.Opts{})
+	require.NoError(t, err)
+
+	b := box.New(conn)
+
+	// Create new user
```

Comment:
Please, checkk all comments for dots )
```suggestion
	// Create new user.
```